common-utils/src/main/java/eu/arrowhead/common/mqtt/handler/MqttTopicHandler.java

```diff
@@ -175,14 +175,14 @@ protected <T> T readPayload(final Object payload, final TypeReference<T> dtoType
 	}
 
 	//-------------------------------------------------------------------------------------------------
-	private Entry<String, MqttRequestModel> parseMqttMessage(final MqttMessage message) {
-		if (message == null) {
+	private Entry<String, MqttRequestModel> parseMqttMessage(final MqttMessageContainer msgContainer) {
```

Comment:
can msgContainer be null?

Reply:
It can't be null. MqttDispatcher always instantiates an MqttMessageContainer, but I add an assert here just to be stay in safe side. Who knows what will change in the future...